src/jit/simd.cpp

```diff
@@ -2622,6 +2628,46 @@ GenTreePtr Compiler::impSIMDIntrinsic(OPCODE                opcode,
         }
         break;
 
+        case SIMDIntrinsicNarrow:
+        {
+            assert(!instMethod);
+            op2 = impSIMDPopStack(simdType);
+            op1 = impSIMDPopStack(simdType);
+
+            simdTree = gtNewSIMDNode(simdType, op1, op2, simdIntrinsicID, baseType, size);
+            retVal   = simdTree;
+        }
+        break;
+
+        case SIMDIntrinsicWiden:
+        {
+            GenTree* dstAddrHi = impSIMDPopStack(TYP_BYREF);
+            GenTree* dstAddrLo = impSIMDPopStack(TYP_BYREF);
+            op1                = impSIMDPopStack(simdType);
+            GenTree* dupOp1    = fgInsertCommaFormTemp(&op1, gtGetStructHandleForSIMD(simdType, baseType));
+
+            // Widen the lower half and assign it to dstAddrLo.
+            simdTree = gtNewSIMDNode(simdType, op1, nullptr, simdIntrinsicID, baseType, size);
```

Comment:
>simdTree = gtNewSIMDNode(simdType, op1, nullptr, simdIntrinsicID, baseType, size); [](start = 12, length = 82)

We are re-using SIMDIntrinsicWiden to mean two different things depending on the context: prior to importation to stand the intrinsic and after importation to stand for widening the lower-half.

What we are saving is one adding one additional enum, which is not a big saving in IMO.  I would suggest to add SIMDIntrinsicWidenLow to avoid any confusion.